### ISSUE TITLE
Adding build time option to dump decompositions

### ISSUE THREAD

**Issue by jayeshkrishna** [ACME-Climate/ParallelIO#13]
Adding a build time option (PIO_SAVE_DECOMPS) that enables
automatic dumping of decompositions (whenever user calls
pio_initdecomp() ).

**Comment by jayeshkrishna** [comment]
@dqwu : Was this feature added in 8d88d2887f6dc342e3c9c2e3984aa32f3052c95b ?

**Comment by dqwu** [comment]
@jayeshkrishna 
Yes, this feature is already in master branch so we can close this issue, thanks.
